### PR TITLE
RCT/JX/Rule5-41

### PR DESCRIPTION
**Pull request by Jiarongx-Xie**
This rule has been tested.

### CODE REVIEW

File: rct229/rulesets/ashrae9012019/section5/section5rule41.py [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+                    "absorptance_solar_exterior_p": find_one(
+                        "$.optical_properties.absorptance_solar_exterior", roof_p
+                    ),
+                    "absorptance_thermal_exterior_b": find_one(
+                        "$.optical_properties.absorptance_thermal_exterior", roof_b
+                    ),
+                    "absorptance_thermal_exterior_p": find_one(
+                        "$.optical_properties.absorptance_thermal_exterior", roof_p
+                    ),
+                    "surface_conditioning_category_b": scc_dict_b[roof_b["id"]],
```
Comment (weilixu) [inline rct229/rulesets/ashrae9012019/section5/section5rule41.py:96]: Remove this `surface_conditioning_category_b`. It is not not needed.

File: rct229/rulesets/ashrae9012019/section5/section5rule41.py [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+        class RoofRule(RuleDefinitionBase):
+            def __init__(self):
+                super(Section5Rule41.BuildingRule.RoofRule, self).__init__(
+                    rmrs_used=produce_ruleset_model_instance(
+                        USER=False, BASELINE_0=True, PROPOSED=True
+                    ),
+                    required_fields={
+                        "$": ["optical_properties"],
+                    },
```
Comment (weilixu) [inline rct229/rulesets/ashrae9012019/section5/section5rule41.py:76]: I found this is the same as `absorptance_solar_exterior` or any of these data is missing. 
I suggest removing the required data field here.

**Review by weilixu — APPROVED** [review]
Approved with two comments need to address.

File: rct229/rulesets/ashrae9012019/section5/section5rule41.py (hunk 1 above)
Comment (Jiarongx-Xie) [reply rct229/rulesets/ashrae9012019/section5/section5rule41.py:96]: Good catch! removed

File: rct229/rulesets/ashrae9012019/section5/section5rule41.py (hunk 2 above)
Comment (Jiarongx-Xie) [reply rct229/rulesets/ashrae9012019/section5/section5rule41.py:76]: Addressed. Thank you for double-checking!

**Comment by Jiarongx-Xie** [comment]
Thank you @weilixu for reviewing this. I have addressed your comments.

**Review by weilixu — APPROVED** [review]
Excellent work. Thank you Jiarong.

**Pull request merged**